code/modules/projectiles/projectile/special/curse.dm

```diff
@@ -8,10 +8,10 @@
 	hitsound = 'sound/effects/curse4.ogg'
 	layer = LARGE_MOB_LAYER
 	damage_type = BURN
-	damage = 10
-	paralyze = 20
+	damage = 0
+	paralyze = 0
```

Comment:
if the curse hands don't do anything, why fire them?

they should at least knock you down or something